editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -4362,16 +4364,7 @@ bool Node3DEditorViewport::_create_instance(Node *parent, String &path, const Po
 
 	Node3D *node3d = Object::cast_to<Node3D>(instantiated_scene);
 	if (node3d) {
-		Transform3D gl_transform;
-		Node3D *parent_node3d = Object::cast_to<Node3D>(parent);
-		if (parent_node3d) {
-			gl_transform = parent_node3d->get_global_gizmo_transform();
-		}
-
-		gl_transform.origin = preview_node_pos;
-		gl_transform.basis *= node3d->get_transform().basis;
-
-		undo_redo->add_do_method(instantiated_scene, "set_global_transform", gl_transform);
```

Comment:
Removing this code ends up breaking the drag to instantiate scene at cursor position functionality. See mentioned issue.

Reply:
I cannot seem to replicate this.
![placement](https://github.com/godotengine/godot/assets/13228932/c857b5e2-b494-40e0-b7a7-07050d1ef5d1)

Reply:
@MajorMcDoom you seem to be testing with dev2. Do you get the same results when testing with latest master?

Reply:
Unless I'm misunderstanding something, I can see the build in that video is 4.3dev2. Your PR was merged less than a day ago so it's not going to be in any dev build. Can you test after building from the last commit in your PR?

Reply:
Well that's embarrassing, I clicked the wrong exe. 😅
I now have a reliable repro, and I will take a look soon. Thank you!

Reply:
All good. I'm not seeing a replacement for the code that was deleted, at the very least I would expect that the position of the new instance gets assigned by preview node's position at some point.

Reply:
@clayjohn @ryevdokimov Sorry, noob question: Are there special protocols for modifying a PR after it's already been merged? Is it possible to work from here, or should I start a new PR?

Reply:
You would just start a new PR.